x-pack/plugins/profiling/common/calculate_impact_estimates/index.ts

```diff
@@ -7,13 +7,19 @@
 
 const ANNUAL_SECONDS = 60 * 60 * 24 * 365;
 
-// The assumed amortized per-core average power consumption.
-const PER_CORE_WATT = 40;
+// The assumed amortized per-core average power consumption (based on 100% CPU Utilization).
+// Reference: https://www.cloudcarbonfootprint.org/docs/methodology/#appendix-i-energy-coefficients
+const PER_CORE_WATT = 7;
 
-// The assumed CO2 emissions per KWH (sourced from www.eia.gov)
-const CO2_PER_KWH = 0.92;
+// The assumed CO2 emissions in kg per kWh.
+// Reference: https://www.cloudcarbonfootprint.org/docs/methodology/#appendix-v-grid-emissions-factors
+const CO2_PER_KWH = 0.379069;
```

Comment:
@rockdaboot  - could you please add a comment here to indicate that this value represents "regional carbon intensity" and it defaults to `us-east-1` ?  It might be best to rename the parameter if you can - to help the team (@cauemarcondes ?) make sense of the calculations in the future. 

 Question,  I am assuming that you multiplied the original value of 0.000379069 metric ton/kWh by 1000 since your calculation is based on CO2 emission in Kg per kwh, is that right?

Reply:
Good idea to clarify the value's origin (commit pushed).

> I am assuming that you multiplied the original value of 0.000379069 metric ton/kWh by 1000 since your calculation is based on CO2 emission in Kg per kwh, is that right?

Yes, exactly. I slightly amended the comment to tell the reader that the references use metric tons/kWh.

Reply:
Looks good!